core/const.hpp

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+namespace fc {
+  constexpr auto kBlockGasLimit{10000000000};
```

Comment:
Probably "primitives/block/block.hpp" is a better place for block related constants.